src/pkg/blob/manager.go

```diff
@@ -112,10 +117,18 @@ func (m *manager) Update(ctx context.Context, blob *Blob) error {
 	return m.dao.UpdateBlob(ctx, blob)
 }
 
+func (m *manager) UpdateBlobStatus(ctx context.Context, blob *models.Blob) (int64, error) {
```

Comment:
if the status is not updated due to version or status mismatch ,it should fail early and explicitly?

Reply:
the first return value can be used to judge update succcess status, as the head blob middleware does.